Makefile

```diff
@@ -0,0 +1,59 @@
+# Define variables
+ENV_FILE=.env
+ENV=local
+
+# Default target
+all: setup
+
+# Setup the application
+setup: install_deps set_env generate_env collect_static migrate setup_waffle_switches
+
+# Install dependencies
+install_deps:
+	poetry install
```

Comment:
this relies on poetry and npm being installed - might want to check that along with chromedriver.

I suppose a simple way of dealing with this would just be to exit with a message that says to install these things.